frontend/src/pages/Login/index.jsx

```diff
@@ -18,7 +18,7 @@ class Login extends Component {
         <Button
           className="login-button"
           color="link"
-          href={`${API_URI}/api/auth/login`}
+          href={`${API_URI}auth/login`}
```

Comment:
just checking.. no `/` between `${API_URI}` and `auth`?

Reply:
Good catch. Will fix to have no trailing slash in the URI since it's against spec.